adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -275,7 +275,17 @@ private AuthenticationResult tryAcquireTokenSilent(final AuthenticationRequest a
 
         if (shouldTrySilentFlow(authenticationRequest)) {
             Logger.v(TAG, "Try to acquire token silently, return valid AT or use RT in the cache.");
-            authenticationResult = acquireTokenSilentFlow(authenticationRequest);
+            try {
```

Comment:
I think i explained it in the slack channel. I was going to release the contacts permission change and found bugs when testing it. For all the bugs found during the testing should go to release branch and once all the validations are done, we merge it back. 

For your change, it's something came later and it's not on top of the release branch. That's why I'm asking you to send it to dev and we squash it, we'll cherry pick the change from dev to release branch. 

Does it make sense?